guppylang/prelude/builtins.py

```diff
@@ -57,13 +56,12 @@
 L = guppy.type_var("L", linear=True)
 
 
-def py(*_args: Any) -> Any:
+def py(*args: Any) -> Any:
     """Function to tag compile-time evaluated Python expressions in a Guppy context.
 
-    This function throws an error when execute in a Python context. It is only intended
-    to be used inside Guppy functions.
+    This function acts like the identity when execute in a Python context.
     """
-    raise GuppyError("`py` can only by used in a Guppy context")
+    return tuple(args)
```

Comment:
We can no longer throw an error here since `py(...)` expressions are actually evaluated by the interpreter when they occur in function signatures